hot-reload.js

```diff
@@ -0,0 +1,36 @@
+const filesInDirectory = dir => new Promise (resolve =>
```

Comment:
please make this a .ts file, put it under src, and add it to `entry` in `webpack.dev.js` instead of copying it in `build.sh`

Reply:
Removed this file and replaced with `webpack-extension-reloader`